misc/psa-claims-profile-1.json

```diff
@@ -1,11 +1,11 @@
 {
-  "legacy-profile": "PSA_IOT_PROFILE_1",
-  "partition-id": 1,
-  "security-life-cycle": 12288,
-  "implementation-id": "UFFSU1RVVldQUVJTVFVWV1BRUlNUVVZXUFFSU1RVVlc=",
-  "boot-seed": "3q2+796tvu/erb7v3q2+796tvu/erb7v3q2+796tvu8=",
-  "hardware-version": "1234567890123",
-  "software-components": [
+  "psa-profile": "PSA_IOT_PROFILE_1",
+  "psa-partition-id": 1,
+  "psa-security-life-cycle": 12288,
+  "psa-implementation-id": "UFFSU1RVVldQUVJTVFVWV1BRUlNUVVZXUFFSU1RVVlc=",
+  "psa-boot-seed": "3q2+796tvu/erb7v3q2+796tvu/erb7v3q2+796tvu8=",
+  "psa-hardware-version": "1234567890123",
```

Comment:
this is psa-hwver (please double check with the annotations in the P1Claims struct in psatoken/claims_p1.go)

Reply:
Yes checked and corrected!